### PR TITLE
[Feature](functions) support ignore and nullable functions

### PR DESCRIPTION
**Pull request by zclllyybb**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

File: docs/zh-CN/docs/sql-manual/sql-functions/debug-functions/non-nullable.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+-->
+
+## non_nullable
+
+### description
+#### Syntax
+
+`non_nullable(expr)`
+
+如果`expr`为nullable类型，则返回值为该表达式的非nullable值；否则报错。
```
Comment (morrySnow) [inline docs/zh-CN/docs/sql-manual/sql-functions/debug-functions/non-nullable.md:34]: 为啥要报错呢？不变就可以了吧。另外，如果该列中有null如何处理？我看下面把null变成0，这是不可接受的行为，会导致数据错误

**Comment by zclllyybb** [comment]
run buildall

**Review by BiteTheDDDDt — DISMISSED** [review]
LGTM

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Review by zhangstar333 — APPROVED** [review]

**Review by BiteTheDDDDt — APPROVED** [review]

**Pull request merged**